llvm/test/Feature/Repo/repo_alias_aliasOfAlias.ll

```diff
@@ -7,17 +7,27 @@
 
 target triple = "x86_64-pc-linux-gnu-repo"
 
-
-;CHECK: declare void @A()
+;CHECK-NOT: @A = alias void ()
```

Comment:
Could you explain the changes to this file, please?

Reply:
Since the DCE pass is added after pruning pass, the function `A`, `B`, `C` and `D` will be deleted. There is no definition and no declaration in the IR code. Therefore, I  changed test to check **NO** alias definition and **NO** declaration.